### ISSUE TITLE
Train/Test/Validation split

### ISSUE THREAD

**Issue by mbellitti** [mbellitti/wikiart-classifier#15]
I'll implement the naive split: divide the data into train/test, and then further divide the train data in actual train and validation. 
Scikit-learn has a some way of doing k-fold cross validation, but I don't know if it plays nicely with the data generators from Keras. We should check how bad is the interoperability.

**Comment by mbellitti** [comment]
The training/validation set split was easy, it's implemented in `ImageDataGenerator`. The script Mohit wrote now uses it.
